### ISSUE TITLE
Storybook 6.0 Support

### ISSUE THREAD

**Issue by 00cc00** [whitespace-se/storybook-addon-html#24]
Hello 👋 
Today Storybook 6.0 was released. Unfortionately that breaks this package.
Would it be possible to support the new Storybook release?

This is the error I get:
```
vendors~main.5a37531269627f3bf6b9.bundle.js:65160 Uncaught TypeError: Cannot set property 'inside' of undefined
    at vendors~main.5a37531269627f3bf6b9.bundle.js:65160
    at cssExtras (vendors~main.5a37531269627f3bf6b9.bundle.js:65203)
    at Refractor.register (vendors~main.5a37531269627f3bf6b9.bundle.js:61842)
    at Object../node_modules/@whitespace/storybook-addon-html/node_modules/react-syntax-highlighter/node_modules/refractor/index.js (vendors~main.5a37531269627f3bf6b9.bundle.js:62071)
    at __webpack_require__ (runtime~main.99691078705b39185f99.bundle.js:80)
    at Module../node_modules/@whitespace/storybook-addon-html/node_modules/react-syntax-highlighter/dist/esm/prism.js (vendors~main.5a37531269627f3bf6b9.bundle.js:30909)
    at __webpack_require__ (runtime~main.99691078705b39185f99.bundle.js:80)
    at Module../node_modules/@whitespace/storybook-addon-html/node_modules/react-syntax-highlighter/dist/esm/index.js (vendors~main.5a37531269627f3bf6b9.bundle.js:30715)
    at __webpack_require__ (runtime~main.99691078705b39185f99.bundle.js:80)
    at Object../node_modules/@whitespace/storybook-addon-html/lib/SyntaxHighlighter.js (vendors~main.5a37531269627f3bf6b9.bundle.js:27915)
```

My last working version was storybook/html 5.3.19

**Comment by hyunchulkwak** [comment]
@00cc00 This seems related recent dependency updates: https://github.com/whitespace-se/storybook-addon-html/commit/a5c8bd645415e17ac87cce7702c5a11908b2a583

Changing back to v1.2.1 worked for me with storybook v6. This is from my config, FYI.

```
// package.json
"devDependencies": {
  ...
    "@whitespace/storybook-addon-html": "1.2.1",

// .storybook/main.js
module.exports = {
  "stories": [
    "../src/**/*.stories.@(js|jsx|ts|tsx)"
  ],
  "addons": [
    "@whitespace/storybook-addon-html/register",
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/addon-a11y"
  ],

// .storybook/preview.js
import { withHTML } from '@whitespace/storybook-addon-html/react';

export const parameters = {
  html: {
    prettier: {
      tabWidth: 2,
      useTabs: false,
      printWidth: 140,
      htmlWhitespaceSensitivity: 'ignore'
    }
  }
}

export const decorators = [
  withHTML
];
```

**Comment by Aberen** [comment]
@hyunchulkwak Rolling back to 1.2.1 seems to do the trick for me as well! Thanks!

**Comment by radum** [comment]
I am still getting the same error 😢

**Comment by jeanfredrik** [comment]
Ok, I'll have a look

**Comment by daniseguraf** [comment]
I'm on Storybook 6.0.16 with storybook-addon-html@1.2.1, the solution of @hyunchulkwak is working great, thanks!

**Comment by docrinehart** [comment]
Using SB6 and ran into this issue with storybook-addon-html@2.0.0.

Rolling back to 1.2.1 also resolved it for me.

**Comment by jeanfredrik** [comment]
The error was apparently caused by react-syntax-highlighter version 13 and had nothing to do with Storybook 6. I’ve downgraded to version 12 now and published version 2.0.1 of the addon to npm now.
